src/Servers/IIS/test/testassets/InProcessWebSite/Startup.cs

```diff
@@ -347,10 +347,10 @@ private async Task ReadAndWriteEchoLines(HttpContext ctx)
             await ctx.Response.Body.FlushAsync();
 
             var reader = new StreamReader(ctx.Request.Body);
-            while (!reader.EndOfStream)
+            while (true)
             {
                 var line = await reader.ReadLineAsync();
-                if (line == "")
+                if (line == "" || line == null)
```

Comment:
Nit: Any empty string just indicates there was an empty line, not the end of the request body. I think it's correct to only check for null if you're trying to echo the body.

Reply:
Maybe its silly question - why you didnt use `string.IsNullOrEmpty` ?